plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -70,6 +73,7 @@ public void createQueryRunner()
                 "hive.metastore-refresh-interval", "10m"));
         queryRunner.execute(ADMIN, "CREATE SCHEMA " + SCHEMA);
         queryRunner.execute("CREATE TABLE test (test INT)");
+        fileHiveMetastore = FileHiveMetastore.createTestingFileHiveMetastore(temporaryDirectory);
```

Comment:
I think you can create this object in test + comment why we need it (e.g. external metadata change)

Reply:
Moved.